app/page.tsx

```diff
@@ -16,10 +19,15 @@ export default function IndexPage() {
         <h1 className="text-3xl font-extrabold leading-tight tracking-tighter sm:text-3xl md:text-5xl lg:text-6xl">
           A simple TOIL calculator
         </h1>
-        <p className="text-lg text-muted-foreground sm:text-xl">
-          Define rulesets for how your TOIL is calculated, calculate your TOIL
-          and share it with anyone all through the URL
-        </p>
+        <p className="text-lg text-muted-foreground sm:text-xl"><TypeIt options={{
+          speed: 60,
```

Comment:
I tested it out and it seems a bit slow, maybe let's try a higher value as I had to really wait for it to finish